tests/unit/tasks/test-test.js

```diff
@@ -34,7 +38,33 @@ describe('test task test', function() {
         expect(defaultOptions.middleware).to.deep.equal(['middleware1', 'middleware2']);
         expect(defaultOptions.test_page).to.equal('http://my/test/page');
         expect(defaultOptions.config_dir).to.be.an('string');
+      },
+    });
+
+    subject.run({
+      host: 'greatwebsite.com',
+      port: 123324,
+      reporter: 'xunit',
+      outputPath: 'blerpy-derpy',
+      testemDebug: 'testem.log',
+      testPage: 'http://my/test/page',
+      configFile: 'custom-testem-config.json',
+      ssl: false,
```

Comment:
We don't receive default options in our test mocks, so we need to provide them. This tests one branch of the new bit of logic.